lib/mix/test/fixtures/deps_status/deps/ok/mix.exs

```diff
@@ -5,4 +5,8 @@ defmodule Ok.Mixfile do
     [app: :ok,
      version: "0.1.0"]
   end
+
+  def application do
+    [extra_applications: [:logger]]
```

Comment:
With this we're basically testing for free that if a dep depends on `:logger` than we'll include it in the escript (otherwise it would fail). Do you mean we need a test that tests that it indeed fails?